regen-network/regen-web
# feat: APP-188 delete post

## Description

https://regennetwork.atlassian.net/browse/APP-188
needs: https://github.com/regen-network/regen-server/pull/477



---

### Author Checklist

_All items are required. Please add a note to the item if the item is not applicable and
please add links to any relevant follow up issues._

I have...


### How to test



From https://deploy-preview-2409--regen-marketplace.netlify.app/, log in and go to a project page you're an admin of with some existing posts.
Try to delete a post from the data stream section. The post should be removed from the timeline.
Access one of the posts page and delete the post from there. It should redirect to the project data stream section.

### Reviewers Checklist

_All items are required. Please add a note if the item is not applicable and please **add
your handle next to the items reviewed if you only reviewed selected items**._

I have...
